style: apply /style-guide pass to models/app (#2679)
wandb/docs: models/app/features/panels.mdx

| <img src="/images/app_ui/manual_workspace.svg" alt="manual workspace icon" width="32px"  /> | **Manual workspaces** start as blank slates and display only the panels you intentionally add. Choose a manual workspace:<ul><li>When you care mainly about a fraction of the keys logged in the project.</li><li>For more focused analysis.</li><li>To improve the performance of a workspace by avoiding loading panels that are less useful to you.</li></ul>Use [Quick add](#quick-add) to quickly populate a manual workspace and its sections with useful visualizations. |

To change how a workspace generates panels, [reset the workspace](#reset-a-workspace).

<Note>
**Undo changes to your workspace**

To undo changes to your workspace, click the Undo button (arrow that points left) or press `Cmd+Z` (macOS) or `Ctrl+Z` (Windows or Linux).
</Note>

## Reset a workspace

> Resetting a workspace lets you change its mode or clear its panels to start fresh.

Reviewer: ```suggestion
Reset a workspace to change its mode or clear its panels to start fresh.
```
Consider opportunities to avoid gerunds and use more direct language

### Configure a section's layout

To configure the layout of a section, click its gear icon, then click **Display preferences**. The following settings are available:

- **Turn on or off colored run names in tooltips** (turned on by default)
- **Only show highlighted run in companion chart tooltips** (turned off by default)
- **Number of runs shown in tooltips** (a single run, all runs, or **Default**)
- **Display full run names on the primary chart tooltip** (turned off by default)

## View a panel in full-screen mode

> Full-screen mode gives a panel more space and higher-fidelity sampling, which is useful for closer inspection of your data. In full-screen mode, the run selector displays and panels use full-fidelity sampling mode plots with 10,000 buckets, rather than the usual 1,000.

Reviewer: I'm not sure if sampling is higher anymore.

    <Frame>
    <img src="/images/app_ui/panel_fullscreen.png" alt="Full-screen panel"  />
    </Frame>

When you [share the panel](#share-a-panel) while viewing it in full-screen mode, the resulting link opens in full-screen mode automatically.

When you work in full-screen mode, use the following tips:

- To return to a panel's workspace from full-screen mode, click the left-pointing arrow at the top of the page.
- To navigate through a section's panels without exiting full-screen mode, use either the **Previous** and **Next** buttons below the panel or the left and right arrow keys.

> - To reclaim more space for the panel, minimize the run selector with `Cmd+.` (macOS) or `Ctrl+.` (Windows or Linux).

Reviewer: Do we want to use code font for key caps?

- When you view an image from a [media panel](/models/app/features/panels/media) in full-screen mode, keyboard shortcuts can zoom in or out, reset zoom, or zoom to fit. See [Keyboard shortcuts](/models/app/keyboard-shortcuts#media-panels).

See [Keyboard shortcuts](/models/app/keyboard-shortcuts) for other full-screen and panel shortcuts.

## Add panels

The following sections describe ways to add panels to your workspace.

### Add a panel manually

> Adding a panel manually lets you choose exactly which type of visualization to create and configure its settings before it appears in the workspace. Add panels to your workspace one at a time, either globally or at the section level.

Reviewer: I think this might have been better before

To start adding a panel, do one of the following:

- To add a panel globally, click **Add panels** in the control bar near the panel search field.
- To add a panel directly to a section instead, click the section's **action (<Icon icon="ellipsis" iconType="solid"/>)** menu, then click **+ Add panels**.

Then, complete the following steps:

1. Select the type of panel to add, such as a chart. The panel's configuration details appear, with defaults selected.

> 1. Optional: Customize the panel and its display preferences. Configuration options depend on the type of panel you select. To learn more about the options for each type of panel, refer to the relevant section, such as [Line plots](/models/app/features/panels/line-plot/) or [Bar plots](/models/app/features/panels/bar-plot/).

Reviewer: Ditto

</Note>

To open **Quick add**, do one of the following:

- To use **Quick add** to add a panel globally, click **Add panels** in the control bar near the panel search field, then click **Quick add**.
- To use **Quick add** to add a panel directly to a section, click the section's **action (<Icon icon="ellipsis" iconType="solid"/>)** menu, click **Add panels**, then click **Quick add**.

A list of panels appears. Each panel with a checkmark is already included in the workspace. To add panels from the list:

- To add all available panels, click the **Add `[N]` panels** button at the top of the list. The **Quick Add** list closes and the new panels display in the workspace.
- To add an individual panel from the list, hover over the panel's row, then click **Add**. Repeat this step for each panel you want to add, then click the **X** at the top right to close the **Quick Add** list. The new panels display in the workspace.

Optional: Customize the panel's settings.

## Share a panel

> Sharing a panel lets you send a focused view of a specific visualization to collaborators without requiring them to navigate the full workspace. The following procedure shows how to share a panel using a link.

Reviewer: ```suggestion
Share a panel to send collaborators directly to a focused view of a specific visualization without navigating to the full workspace. To share a panel using a link:
```
